test/test_goodmans_er.py

```diff
@@ -1,6 +1,76 @@
 """Test Goodmans Ecological Regression."""
+import numpy as np
+import pytest
 from pyei.goodmans_er import *  # pylint:disable=wildcard-import,unused-wildcard-import
 
 
-def test_true():
-    assert True
+@pytest.fixture
+def group_and_vote_fractions():
+    """ Sample group and vote fractions, where every member of the demographic
+        group votes for the given candidate and every non-member of the
+        demographic group does not vote for the given candidate.
+    """
+    group_share = np.array([0, 0.2, 0.4, 0.6, 0.8])
+    vote_share = np.array([0, 0.2, 0.4, 0.6, 0.8])
+    return group_share, vote_share
+
+
+@pytest.fixture
+def group_and_vote_fractions_with_pop():
+    """ Sample group and vote fractions, where every member of the demographic
+        group votes for the given candidate and 10% of the demographic group's
+        complement supports the given candidate (i.e., slope = 1, intercept = 0.1),
+        with an exception of one precinct.
+        All precincts have population 1000, except one precinct that has population
+        1 and does not follow the above formula.
+    """
+    group_share = np.array([0, 0.2, 0.4, 0.6, 0.8, 0.9])
+    vote_share = np.array([0.1, 0.3, 0.5, 0.7, 0.9, 0.9])
+    populations = np.array([1000, 1000, 1000, 1000, 1000, 1])
+    return group_share, vote_share, populations
+
+
+def test_fit(group_and_vote_fractions):  # pylint:disable=redefined-outer-name
```

Comment:
you can move this pylint line to the top of the file (I think it applies to everything after it in the same scope, so line 2 works), and not worry about it on a per-function basis. 

I _sort_ of want to disable it, but this sort of warning has been helpful to me in the past -- if you want to go that route, feel free to add it to the .pylintrc